shell/common/rasterizer.h

```diff
@@ -257,8 +257,8 @@ class Rasterizer final : public SnapshotDelegate {
   ///
   /// @param[in]  pipeline  The layer tree pipeline to take the next layer tree
   ///                       to render from.
-  ///
-  void Draw(fml::RefPtr<Pipeline<flutter::LayerTree>> pipeline);
+  /// TODO(kaushikiska) fix docs whereever pipeline is mentioned.
```

Comment:
Don't forget this. Probably time to fix the docs as the rest looks pretty stable...